packet/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -209,17 +210,20 @@ storage:
           # https://www.kernel.org/doc/Documentation/admin-guide/devices.txt
           major_numbers="8,259"
 
+          # XXX: These options are exclusive, as only one fs can be mounted
+          # to /mnt/
+          # This is, partly, because when creating dirs inside /mnt to mount
+          # several paths (like /mnt/node-local-storage), those are not visible
+          # to the pods. See this issue for more info:
+          # https://github.com/kinvolk/lokomotive-kubernetes/issues/73
+          #
           # Variables replaced by Terraform
           if [ ${setup_raid} = true ]; then
             create_data_raid "$${major_numbers}" -1 /dev/md/node-local-storage true
-          else
-             # Both can be set independently
-             if [ ${setup_raid_hdd} = true ]; then
-               create_data_raid "$${major_numbers}" 1 /dev/md/node-local-hdd-storage true
-             fi
-             if [ ${setup_raid_ssd} = true ]; then
-               create_data_raid "$${major_numbers}" 0 /dev/md/node-local-ssd-storage ${setup_raid_ssd_fs}
-             fi
+          elif [ ${setup_raid_hdd} = true ]; then
```

Comment:
I am not really sure, why this change was introduced?

With this code now we can only set RAID of type HDD or SSD. This change has added another regression!

Reply:
Because mounting hdd and sdd both on `/mnt` is not possible, as the patch was doing (if you configure both to have an fs. This is explained in the updated PR description :-)

And those changes never worked, they never wrote to those devices, as things mounted under `/mnt` are not visible to pods. I've created #74 to double check if we really need this and also created #73 to tackle the root cause and even proposed a fix there (if better fixes can't be found), see the comment that links to https://github.com/kinvolk/lokomotive-kubernetes/commit/ba802658a17c030ce48d0317b85de901d638a26e

However, as far as I understand those never worked and my reasoning was simple: as those variables never worked, introduced for a few commits a backwards incompatible change and I'm not really sure we need them, let's just revert to the previous working situation and if we need to introduce them and do a backwards incompatible change, let's just do it. But it would be great if we can avoid using these, as the code is getting out of control (also created  #75 to suggest a path to keep this under control) and introducing more bugs than solutions (as just created bugs but never really worked :-D)